CommandCollections.xml

```diff
@@ -194,6 +194,6 @@
 			<name>Generic</name>
 			<name>Online</name>
 		</commandLists>
-		<streamStatus>any</streamStatus>
+		<streamStatus>online</streamStatus>
```

Comment:
Compare to similar command collections in Joshimuz's commands: https://github.com/Joshimuz/BotimuzCommands/blob/da7eb30bae7d256d0aa12fe0299bbe5df527f464/CommandCollections.xml#L353-L371